src/pages/LandingPage.js

```diff
@@ -75,15 +75,16 @@ const LandingPage = () => {
   const [billingProjects, setBillingProjects] = useState()
   const signal = useCancellation()
 
-  useEffect(() => {
-    const loadProjects = withErrorIgnoring(async () => {
-      const projects = await Ajax(signal).Billing.listProjects()
-      setBillingProjects(projects)
-    })
-    if (isSignedIn) {
-      loadProjects()
-    }
-  }, [isSignedIn, setBillingProjects, signal])
+  const loadProjects = withErrorHandling(async error => {
```

Comment:
@yunip20 you lost the surrounding `useEffect`. Sorry for the confusion… I was just providing the updated body of `useEffect`.